utils/training_loop.py

```diff
@@ -13,42 +13,47 @@ def training_loop(experiment, env, agent, model):
         print('Only Inference')
         model = PPO.load(experiment.LOAD_WEIGHT_DIRECTORY)
```

Comment:
The inference needs to be according to the model that we trained on (PPO/A2C/DQN)

Reply:
Done